### DISCUSSION TITLE
Congratulations on React Summit Award!

### DISCUSSION THREAD

**Discussion by ssteuteville** [react-hook-form/react-hook-form#3183] [Show and tell]
Just wanted to share my enthusiasm for the fact that you guys go the reward! A year ago when we started using react-hook-form on version 2.x.x we weren't sure if it was going to work out.. But you have done so much great development over the past year, we have no more doubts! On top of the development, we love the support u give too!

Thanks for all the hard work, you really make our lives easier.

**Comment by bluebill1049** [comment]
Thanks so much, @ssteuteville that's awesome to hear! I am totally surprised by the award and grateful for everyone's support and love.

**Reply by bluebill1049** [reply]
here is the video: https://youtu.be/JW4ws0y1IhU?t=431 still excited watching it.
